app/renderer/components/navigation/urlBar.js

```diff
@@ -233,7 +233,7 @@ class UrlBar extends React.Component {
       return true
       // this is if the new value normalized IS CONTAINED within the suggestion, then just set the url input
       // instead of just updating the suffix
-    } else if (this.props.normalizedSuggestion.includes(newValueNormalized) && this.props.normalizedSuggestion.length > 0) {
+    } else if (this.props.normalizedSuggestion.includes(newValueNormalized) && this.props.normalizedSuggestion.length > 0 && this.props.activeIndex !== null) {
```

Comment:
After the recent master UI updates, this condition had to be added so that if the suggestion simply _included_ the url, it would only fill the url bar when a suggestion index is selected.

This can be avoided if #14433 ends up getting merged, since the first suggestion would always be what the user is typing in.